lms/djangoapps/sneakpeek_deeplink/tests.py

```diff
@@ -63,6 +63,13 @@ def setUp(self):
             enrollment_start=now - month2,
             enrollment_end=now - month)
         self.closed_course.save()
+        self.course_to_delete = CourseFactory.create(
+            org='deleted',
+            number='course',
+            run='run',
+            enrollment_start=now - month,
+            enrollment_end=now + month)
```

Comment:
Should I apply these changes to the course creations that existed before my changes?